### ISSUE TITLE
Transforms ignored on .fa-layers-text when using nest mutator

### ISSUE THREAD

**Issue by idpaterson** [FortAwesome/Font-Awesome#13497]
The nest mutator, used when the `autoReplaceSvg` option is set to `"nest"`, causes power transforms to be ignored on `.fa-layers-text`. The mutator ignores all attributes including the `style` attribute generated by `makeLayersTextAbstract()`.

[This fiddle](https://jsfiddle.net/wLg2xc9p/8) shows the calendar example from the Font Awesome documentation. While this works properly with `autoReplaceSvg: true` (which uses the `replace` mutator), no transform is applied with `autoReplaceSvg: "nest"` and the numbers overlap the entire calendar icon.

```html
<span class="fa-layers fa-3x">
  <i class="fas fa-calendar"></i>
  <span class="fa-layers-text fa-inverse" data-fa-transform="shrink-8 down-3">27</span>
</span>
```

| Expected | `replace` | `nest` |
| --- | --- | --- |
| ![correct calendar](https://user-images.githubusercontent.com/507058/42415925-77759fb0-822e-11e8-99d8-b973988fdd36.png) |  ![correct calendar](https://user-images.githubusercontent.com/507058/42415925-77759fb0-822e-11e8-99d8-b973988fdd36.png) | ![non-transformed calendar](https://user-images.githubusercontent.com/507058/42415928-8245615a-822e-11e8-88f6-5acb41434601.png) | 

I have found the nest mutator very useful in upgrading an open source project from Font Awesome 4 to 5 Pro. This is certainly something that could be worked around by using the default replace mode instead, but since that would require much refactoring of old markup and CSS I am hopeful for a fix to make replace and nest mode produce the same result.

**Comment by robmadole** [comment]
@idpaterson thanks for the great bug report. I can confirm this is broken. We'll get it on the list to get fixed!

**Comment by MajorEvent** [comment]
Just bumping this since it's affecting me and not fixed yet. It's not affecting layering icons or the counter, just the text transform.

**Comment by Chris-KCIT** [comment]
Bump, looking forward to seeing this resolved, or at a minimum mentioned in the documentation if it is going to be an ongoing issue.

**Comment by DeetyB** [comment]
Another bump! Also, the same issue applied to counters (fa-layers-counter) as well as text (fa-layers-text).

**Comment by dbarwikowski** [comment]
Bump

**Comment by amiram** [comment]
Do you know when this is going to be fixed?

**Comment by robmadole** [comment]
We don't have a timeline for this yet. But it's still on the list.

**Comment by Chris-KCIT** [comment]
For me, a work around fix would be icons for numbers between 1 and 31, but would really love for this to be fixed for good.

**Comment by MHedish** [comment]
Bump!

**Comment by sergiopantoja** [comment]
2021 Bump 🥸

**Comment by Chris-KCIT** [comment]
FA6 does seem to have numbers now, it would be way better if these went up to 31 IMO - but even better if this bug could be addressed 😮
![image](https://user-images.githubusercontent.com/9876915/121090947-ab1f3f00-c82c-11eb-9f2e-f16dfe69215a.png)

**Comment by julienlegault** [comment]
Does anyone have a fix for fa-layers-text? The official documentation states there is a workaround but doesn't explain what that might be.

**Comment by tagliala** [comment]
@juliankrispel I think it just suggests to use standard CSS customized for your specific needs

This is a not-official example: https://jsfiddle.net/tagliala/fmo25dxa/17/

**Comment by robmadole** [comment]
We've released a lot of fixes around layers in 6.0.0-beta2. I believe we have this one fixed so I'm going to close.

If anyone comes back around and it's still an issue just let me know and provide a reproducible test case and we'll re-open.

**Comment by jasonlundien** [comment]
A fix was released with 6.0.0-beta2.
